translations/base-de.yaml

```diff
@@ -30,41 +30,41 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io ist ein Spiel über den Bau von Fabriken um die Erstellung und Kombination von Formen zu automatisieren. Liefere die gewünschten, stetig komplexer werdenden Formen, um im Spiel voranzukommen und schalte Upgrades frei, die deine Fabrik zu beschleunigen!
+        In shapez.io umringst du deine Basis mit selbstgebauten Fabriken, die Formen automatisiert Erstellen, Bearbeiten und Kombinieren sollen. Liefere die gewünschten, stetig komplexer werdenden Formen an dein Hauptgebäude, um im Spiel voranzukommen. Schalte mit ihnen außerdem Upgrades frei, die deine Maschinen und somit auch deine Fabriken beschleunigen!
 
-        Da die Nachfrage steigt, wirst du deine Fabrik vergrößern müssen, um den Bedürfnissen gerecht zu werden - vergiss jedoch nicht die Ressourcen, du wirst in der [b]unendlichen Karte[/b] expandieren müssen!
+        Da die Nachfrage sowohl in der Komplexität, als auch der Anzahl steigt, wirst du deine Fabriken erweitern müssen. Vergiss nicht, dass du die dafür benötigten Ressourcen beschaffen musst und expandiere auf der [b]unendlichen Karte[/b]!
 
-        Da Formen natürlich langweilig werden können, musst du Farben mischen und deine Formen damit bemalen - Kombiniere rote, grüne und blaue Farbressourcen, um verschiedene Farben herzustellen und Formen damit zu bemalen, um die Nachfrage zu befriedigen.
+        Das reine Zerschneiden und Zusammensetzen von Formen wird natürlisch schnell langweilig, also musst du Farben mischen und deine Formen damit bemalen lassen. Kombiniere die Farbpartikel, um verschiedene Farbtöne herzustellen. Staple dann deine fertigen Formen aufeinander und lasse die wildesten Kreationen entstehen.
```

Comment:
`natürlisch `

Reply:
uff 😅

Reply:
Dreimal überlegt ob ich es reinnehme und beim vierten Mal falsch geschrieben

Reply:
Wie stark soll denn der Gebrauch von "du" sein? Geht das mit der direkten Ansprache in Ordnung, oder eher nur im Ausnahmefall benutzen?